src/components/aspect/aspect.js

```diff
@@ -30,10 +30,10 @@ export const BAspect = /*#__PURE__*/ Vue.extend({
       const aspect = this.aspect
       let ratio = 1
       if (RX_ASPECT.test(aspect)) {
-        const [width, height] = aspect.split(RX_SEPARATOR).map(v => toFloat(v) || 1)
+        const [width, height] = aspect.split(RX_SEPARATOR).map(v => Math.max(toFloat(v, 0), 1))
         ratio = width / height
       } else {
-        ratio = toFloat(aspect) || 1
+        ratio = Math.max(toFloat(aspect, 0), 1)

```

Comment:
This wont work, as aspects could be less than `1` (i.e. `0.5`)

Only if ratio is `NaN` or `0` we default it to 1